src/parser/priest/discipline/modules/core/AtonementAnalyzer.ts

```diff
@@ -0,0 +1,129 @@
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
```

Comment:
One of the two core drivers behind a lot of the spec functionality, emits AtonementEvents as pairs with a source DamageEvent and a resulting HealEvent. Has a dependency on the Spirit Shell module to emit events with the same contract regardless of final tier talent use.